lib/helpers/verify-component-specification-version.js

```diff
@@ -8,16 +8,6 @@ const { getOrigamiJson } = require('./files');
  */
 module.exports = async () => {
 	const origamiJson = await getOrigamiJson();
-	if (!origamiJson || typeof origamiJson.origamiVersion !== 'number') {
-		throw new Error(
-			'Is this an Origami component? \n\n' +
-			'Origami Build Tools may only be used to develop Origami ' +
-			'components. It looks like this isn\'t an Origami component, ' +
-			'or the component is missing a valid "origami.json". See the ' +
-			'component specification for more details: ' +
-			'https://origami.ft.com/spec/'
-		);
-	}
 	if (origamiJson.origamiVersion === 1) {
```

Comment:
I think `getOrigamiJson` can return a null value 😬 if I remember right 

```suggestion
	if (origamiJson && origamiJson.origamiVersion === 1) {
```